config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/bundle/builder/BundleEntityBuilder.java

```diff
@@ -137,21 +149,35 @@ private Element createDeleteBundle(final Document document, List<Entity> entitie
         return bundleDocumentBuilder.build(document, deleteBundleEntities);
     }
 
+    /**
+     * Creates the DELETE environment bundle element.
+     *
+     * @param document Document
+     * @param entities Entities packaged in the deployment bundle
+     * @return Delete bundle Element for the Annotated Bundle
+     */
+    private Element createDeleteEnvBundle(final Document document, List<Entity> entities) {
+        List<Entity> filteredEntities = copyFilteredEntitiesForDeleteBundle(entities, FILTER_ENV_ENTITIES);
+        filteredEntities.forEach(e -> e.setMappingAction(MappingActions.DELETE));
+        return bundleDocumentBuilder.build(document, filteredEntities);
+    }
+
     /**
      * Copies all the filtered entities in the reverse order to a new {@link List}. The entries in the DELETE bundle
      * must be in the reverse order of deployment bundle.
-     *
+     * <p>
```

Comment:
Whats this? shouldn't we have a closing one as well?

Reply:
removed